app/src/main/java/com/example/androidthings/assistant/AssistantActivity.java

```diff
@@ -179,11 +182,24 @@ public void onSpeechRecognition(List<SpeechRecognitionResult> results) {
                 })
                 .setConversationCallback(new ConversationCallback() {
                     @Override
-                    public void onAudioSample(ByteBuffer audioSample) {
-                        if (mLed != null) {
+                    public void onResponseStarted() {
+                        super.onResponseStarted();
+                        // When bus type is switched, the AudioManager needs to reset the stream volume
+                        if (mDac != null) {
+                            try {
+                                mDac.setSdMode(Max98357A.SD_MODE_LEFT);
+                            } catch (IOException e) {
+                                Log.e(TAG, "error enabling DAC", e);
```

Comment:
did you want to also print e.getMessage() ?

Reply:
Passing in an `Exception` as the last parameter will print the message and stacktrace (which doesn't seem to be true for a `Throwable`)